protocol_check/action.yml

```diff
@@ -0,0 +1,21 @@
+name: "check protocol"
+description: "check the protocol in github.com/inbo/protocolsource"
+inputs:
+  PAT:
+    description: >
+      Personal access token (PAT) used to fetch the repository. The PAT is configured
+      with the local git config, which enables your scripts to run authenticated git
+      commands. The post-job step removes the PAT.
+
+
+      We recommend using a service account with the least permissions necessary.
+      Also when generating a new PAT, select the least scopes necessary.
+
+
+      [Learn more about creating and using encrypted secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets)
+    default: ${{ github.token }}
+runs:
+  using: 'docker'
+  image: docker://inbobmk/protocols
```

Comment:
no entrypoint needed here?

Reply:
No, I don't think so.  The dockerfile in protocolsource refers to this entrypoint (`entrypoint_check.sh`), so this is the default when running the docker.